ocaml/typing/env.ml

```diff
@@ -877,6 +895,21 @@ let md md_type =
   {md_type; md_attributes=[]; md_loc=Location.none
   ;md_uid = Uid.internal_not_actually_unique}
 
+(** The caller is not interested in modes, and thus [val_modalities] is
+invalidated. *)
```

Comment:
Should we also be invalidating the mode? Is it meaningful without knowing the modalities?